app/predacons_cli/src/rag.py

```
@@ -0,0 +1,108 @@
+        print("✅ Done")
+    
+    def load_db(self):
+        db = Chroma(persist_directory=self.chroma_path, embedding_function=self.embedder)
+        return db
+    
+    def get_similar(self, query,db =None, top_n = 5,similarity_threshold = 0.1):
+        if db is None:
+            db = self.load_db()
+        results = db.similarity_search_with_relevance_scores(query, k=top_n)
+        if len(results) == 0 or results[0][1] < similarity_threshold:
+            print(f"Unable to find matching results.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)